madgiraffestudios/src/index.js

```diff
@@ -6,30 +6,24 @@ import App from './App';
 import './App.scss';
 import * as serviceWorker from './serviceWorker';
 import NavigationBar from './components/Navigation/NavigationBar'
-//import About from './components/About/About';
-import { WORK } from './Constants/routes';
+import About from './components/About/About';
+import Home from './components/Home/home';
+import { ABOUT } from './Constants/routes';
 
```

Comment:
Great work, would be nice if we just use one form of for the routes...either way you used "About" or "Home" but not both

Reply:
fixed